src/lib/cli-util.js

```diff
@@ -1,59 +1,59 @@
-var size = require('window-size')
-var availableWidth = size.width || /*istanbul ignore next */ 80
-var ui = require('cliui')({width: availableWidth})
+const size = require('window-size');
+
+const availableWidth = size.width || /* istanbul ignore next */ 80;
+const ui = require('cliui')({width: availableWidth});
 
 function push(output, columns, uniformColWidths) {
-  var _output = [].concat(output)
+  const _output = [].concat(output);
 
-  var padding = {top: 0, right: 2, bottom: 0, left: 0}
-  var maxWidth = [_output.reduce(
-    function getMaxWidth(previous, current) {
-      return Math.max(padding.left + current.length + padding.right, previous)
-    }, 0)]
+  const padding = {top: 0, right: 2, bottom: 0, left: 0};
+  const maxWidth = [_output.reduce((previous, current) => {
+    return Math.max(padding.left + current.length + padding.right, previous);
```

Comment:
```JavaScript
const maxWidth = [_output.reduce((previous, current) => Math.max(padding.left + current.length + padding + padding.right, previous), 0)];
```

Why is `maxWidth`'s value wrapped in an array iteral in the first place? I don't remember.